ui.frontend/src/view/FormFieldBase.js

```diff
@@ -99,7 +99,7 @@ export default class FormFieldBase extends FormField {
             this._updateValue(state.value);
         }
         this._updateVisible(state.visible)
-        this._updateEnabled(state.enabled)
+        this._updateEnabled(this._model.enabled)
```

Comment:
I am not sure why are we doing this. Reading this way, enables dependency tracking which we do not want.

In the core you need to change the enabled property in the state as well. getState() already does that for some properties which are dynamic.

Reply:
We always update the _jsonModel and I understand we do not want that for the enabled property but the getter and the getState() should always return the same value.

Reply:
Ok @vdua , thats what i wanted to understand that whether _jsonModel is meant to always store the values as saved in crx or its values can change during runtime accordingly. 

I will make the change, since we are not actually altering the enabled field in model and only the getter is altering the returned value, should i change the value of _jsonModel.enabled only when getState() is called?
cc:- @vdua , @nit23uec

Reply:
Also @vdua , 
just for my understanding, can you please explain why we shouldn't use this._model.enable, how is it dependency tracking in this way. i am clear that getState and getters should return the same thing, that's what i also discussed yesterday but i guess i understood the solution incorrectly.
Thanks
Deep